### PR TITLE
routed the enums

### PR DESCRIPTION
**Pull request by TinyTani**

### CODE REVIEW

**Comment by VarioCode** [comment]
Please resolve the conflict first.

**Review by jortdr — CHANGES_REQUESTED** [review]
Resolve conflicts

**Comment by jortdr** [comment]
There are still conflicts here.

File: OpenPOS-APP/MenuPage.xaml.cs [hunk 1]
```diff
@@ -57,16 +56,16 @@ public void AddAllProducts()
...
-		{
-			AddHorizontalLayout();
-		}
-        
-		ProductView productView = new ProductView();
-		productView.SetProductValues(this,product);
-		productView.ClickedMoreInfo += OnInfoButtonClicked;
-		HorizontalLayout.Add(productView);
-        Debug.WriteLine(HorizontalLayout.Children.Count);
+       if (HorizontalLayout == null || HorizontalLayout.Children.Count % moduloNumber == 0)
```
Comment (jortdr) [inline OpenPOS-APP/MenuPage.xaml.cs:59]: HorizontalLayout needs to be refactored

**Review by jortdr — CHANGES_REQUESTED** [review]
Check comments

**Review by jortdr — APPROVED** [review]

**Pull request merged**